src/FieldWarning/Assets/Ingame/UI/InputManager.cs

```diff
@@ -11,17 +11,19 @@
  * the License for the specific language governing permissions and limitations under the License.
  */
 
-using UnityEngine.EventSystems;
-using UnityEngine;
-using System.Collections.Generic;
-using System.Linq;
-using System;
-
-using PFW.Ingame.Prototype;
-using PFW.Model.Game;
-
 namespace PFW.Ingame.UI
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Resources;
+
+    using PFW.Ingame.Prototype;
+    using PFW.Model.Game;
+
+    using UnityEngine;
+    using UnityEngine.EventSystems;
+
```

Comment:
The entire codebase puts includes outside the namespace, and most incoming programmers will also be used to that ordering. Since both approaches are largely equivalent, I think it's easier for you to change your stylecop defaults or whatever is pushing you to use this style.